preplan-backend/config/db.config.js

```diff
@@ -8,27 +7,86 @@ const initDB = () => {
                 name: "volunteer"
             }).save(err => {
                 if (err) {
-                    console.log("error", err);
+                    console.log("Error creating role", err);
                 }
             });
 
             new Role({
                 name: "organizer"
             }).save(err => {
                 if (err) {
-                    console.log("error", err);
+                    console.log("Error creating role", err);
                 }
             });
 
             new Role({
                 name: "admin"
             }).save(err => {
                 if (err) {
-                    console.log("error", err);
+                    console.log("Error creating role", err);
                 }
             });
         }
     });
+
+    Skill.estimatedDocumentCount((err, count) => {
+        if (!err && count === 0) {
+
```

Comment:
How are we going to deal with the "Any departments" ? 
Add them as skill in DB and then treat it before sending it to the generator? 
Or Any departments is already as a concat of all skills?

Reply:
Any should be a list of everything